entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+cd rems
+
+[ -z "$COMMANDS" ] && COMMANDS="run"
+
+for COMMAND in $COMMANDS
+do
+    if [ "${COMMAND}" = "run" ] ; then
+        COMMAND=""
+    fi
+    echo "####################"
+    echo "########## RUNNING COMMAND: java --illegal-access=deny -Drems.config=config/config.edn -jar rems.jar ${COMMAND}"
```

Comment:
flags don't match the actual command. perhaps use a variable for the whole command line?